src/js/views/eva-study-view.js

```diff
@@ -143,7 +143,7 @@ EvaStudyView.prototype = {
         var pubLinks = '';
         if(!_.isEmpty(publications) && publications != '-') {
             for (i = 0; i < publications.length; i++) {
-                pubLinks += '<a class="pubmed-id" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=' + publications[i] + '" target="_blank">' + publications[i]  + '</a><br />'
+                pubLinks += '<span class="pubmed-id">' + publications[i]  + '</span><br />'
```

Comment:
It might be worth changing that class's name. I only see it used [here](https://github.com/EBIvariation/eva-web/blob/ec2be8f56e4721f624d97fe0c70566065b4772a9/tests/acceptance/study_view.js#L187) on top of the `eva-study-view.js` and `eva.js`

Reply:
good point, updated